### DISCUSSION TITLE
Table sorting and filtering in database runs

### DISCUSSION THREAD

**Discussion by ghost** [langfuse/langfuse#3646] [Ideas]
### Describe the feature or potential improvement

For a database run, it would be nice to be able to sort the items based on e.g. their scoring like you're able to do in the generations table. E.g.
![image](https://github.com/user-attachments/assets/ff9b16ab-c435-4ccc-91d3-99321496ba12)
Here it would be nice to filter out the correct classifications so it is easier to identify what prompt changes to make.

**Comment by Kitenite** [comment]
This would be really nice. We'd like to sort users and sessions by cost for example

**Reply by Haoping-Xiao** [reply]
@marcklingen please take a look at this feature.

filter empty output to inspect. sort by score/latency for example
